README.adoc

```diff
@@ -599,6 +599,28 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping Runnable Method [[skipping-runnable-method]]
```

Comment:
Sorry for the miss-suggestion. Looking at other rules of RuboCop HQ guides, `methods` might be better.

```suggestion
=== Skipping Runnable Methods [[skipping-runnable-methods]]
```